### DISCUSSION TITLE
Yellowbrick and AutoGluon

### DISCUSSION THREAD

**Discussion by ahessane** [autogluon/autogluon#2938] [Q&A] [answered]
Hello,

Is there a way to use a tabular predictor (i.e, best model) as an estimator for yellowbrick ? I want to plot some high quality plots such as Risiduals, Q-Q Plot, etc. Thank you

**Comment by gradientsky** [comment, marked as answer]
Not at the moment. However, we are working on EDA package, which has some [model analysis tooling](https://auto.gluon.ai/2023_01_25_eda_docs/tutorials/eda/eda-auto-quick-fit.html). Feel free to cut new issues if you see any tools would benefit the users.

**Reply by ahessane** [reply]
Thank you. I look forward to test the new features.

**Reply by gradientsky** [reply]
PR just got merged: https://github.com/autogluon/autogluon/pull/3039
The [next nightly build](https://pypi.org/project/autogluon.eda/0.7.0b20230316/) will have the change. Thanks for a good idea.
